projects/vdk-core/src/vdk/internal/builtin_plugins/config/vdk_config.py

```diff
@@ -83,6 +84,13 @@ def vdk_configure(self, config_builder: ConfigurationBuilder) -> None:
             "Allowed values: CRITICAL, ERROR, WARNING, INFO, DEBUG. "
             "If not set python default or one set by vdk -v LEVEL is used. ",
         )
+        config_builder.add(
+            LOG_STACK_TRACE,
+            False,
+            True,
+            "Controls whether the full stack trace is displayed again on exit code 1. "
+            "False by default, set to True in production environments for more debug output. ",
```

Comment:
> set to True in production environments for more debug output

That is not up to us. But up to the operator of VDK or the end user.

Reply:
It was meant to be an imperative. 😄 As in "Set this to true in prod environments if you want more debug output". I'll change it so it's more clear.